warehouse/templates/accounts/profile.html

```diff
@@ -39,6 +39,12 @@ <h1 class="author-profile__name">{{ user.name }}</h1>
               &nbsp;&nbsp;{% trans start_date=humanize(user.date_joined) %}Joined on {{ start_date }}{% endtrans %}
             </p>
             {% endif %}
+            {% if projects %}
+            <p>
+              <i class="fa fa-rss" aria-hidden="true"></i>
+              <span class="sr-only">{% trans %}RSS Feed{% endtrans %}</span>
+              &nbsp;&nbsp;<a href="{{ request.route_path('rss.user_updates', username=user.username) }}">{% trans %}Recent updates{% endtrans %}</a></p>
+            {% endif %}
```

Comment:
I think adding in the extra `alternate` link probably makes sense, regardless.  I'll add it in with the bugfix.

Reply:
I think it makes sense to make the link visible in the profile somewhere, otherwise it would be challenging to find these feeds. The other site-wide ones are listed at https://warehouse.readthedocs.io/api-reference/feeds/, and this PR should probably update that documentation as well.

Reply:
Done.